helm/openidm/values.yaml

```diff
@@ -33,8 +33,10 @@ image:
 # override Java JVM options.
 javaOpts: "-Xmx1024m -server -XX:+UseG1GC"
 
-# If you set this to true, you must have a TLS secret with the same name as the FQDN.
+# TLS is enabled by default. See TLS section for configuration options
```

Comment:
See my suggestion in the README for replacing this comment.

Reply:
done